evap/staff/tests/test_importers.py

```diff
@@ -13,14 +13,58 @@ class TestUserImporter(TestCase):
     filename_invalid = os.path.join(settings.BASE_DIR, "staff/fixtures/invalid_user_import.xls")
     filename_random = os.path.join(settings.BASE_DIR, "staff/fixtures/random.random")
 
+    # valid user import tested in tests/test_views.py, TestUserImportView
+
+    @classmethod
+    def setUpTestData(cls):
+        with open(cls.filename_valid, "rb") as excel_file:
+            cls.valid_excel_content = excel_file.read()
+        with open(cls.filename_invalid, "rb") as excel_file:
+            cls.invalid_excel_content = excel_file.read()
+        with open(cls.filename_random, "rb") as excel_file:
+            cls.random_excel_content = excel_file.read()
+
+    def test_test_run_consistency(self):
+        original_users = list(UserProfile.objects.all())
+
+        __, __, __, __ = UserImporter.process(self.valid_excel_content, test_run=True)
+
+        self.assertEqual(original_users, list(UserProfile.objects.all()))
+
+    def test_test_and_notest_equality(self):
+        # success messages are supposed to be different
+        list_test, __, warnings_test, errors_test = UserImporter.process(self.valid_excel_content, test_run=True)
+        list_notest, __, warnings_notest, errors_notest = UserImporter.process(self.valid_excel_content, test_run=False)
+
+        notest_string_list = ["{} {}".format(user.full_name, user.email) for user in list_notest]
+        for user in list_test:
+            self.assertIn("{} {}".format(user.full_name, user.email), notest_string_list)
+
+        self.assertEqual(warnings_test, warnings_notest)
+        self.assertEqual(errors_test, errors_notest)
+
+    def test_created_users(self):
+        original_user_count = UserProfile.objects.count()
+
+        user_list, v1, v2, v3 = UserImporter.process(self.valid_excel_content, test_run=False)
+
+        self.assertIn("Successfully read sheet 'Users'.", v1)
+        self.assertIn('Successfully created 2 user(s):<br>Lucilia Manilium (lucilia.manilium)<br>Bastius Quid (bastius.quid.ext)', v1)
+        self.assertIn('Successfully read Excel file.', v1)
+        self.assertEqual(v2, {})
+        self.assertEqual(v3, [])
+
+        self.assertEqual(len(user_list), 2)
+        self.assertEqual(2 + original_user_count, UserProfile.objects.count())
```

Comment:
also, swap the two arguments to `assertEqual`. otherwise it gets close to yoda expressions, `2 == some_variable`